pytest_container/container.py

```diff
@@ -1235,6 +1235,28 @@ def __exit__(
             mounts = self.container_runtime.inspect_container(
                 self._container_id
             ).mounts
+
+            _logger.debug(
+                "Stopping container %s via %s",
+                self._container_id,
+                self.container_runtime.runner_binary,
+            )
+            try:
+                check_output(
+                    [
+                        self.container_runtime.runner_binary,
+                        "stop",
+                        self._container_id,
+                    ]
+                )
+            except Exception as e:
+                _logger.debug(
+                    "Stopping container %s via %s failed, exception: %s, ignoring",
+                    self._container_id,
+                    self.container_runtime.runner_binary,
+                    e,
+                )
```

Comment:
We shouldn't be catching all exceptions, this can mask potentially important issues. Instead, we should be using `call` or `Popen` and optionally logging if the stop fails.
```suggestion
            call(
                [
                    self.container_runtime.runner_binary,
                    "stop",
                    self._container_id,
                ]
            )
```